<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick and Dirty Parser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="serialOutputLayout">
        <item>
         <layout class="QHBoxLayout" name="labelAndButtons">
          <item>
           <widget class="QLabel" name="serialOutputLabel">
            <property name="text">
             <string>Display as:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="displayOutputBit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Bits</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">displayButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="displayOutputHex">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Hex</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">displayButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="displayOutputChar">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Char</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">displayButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="serialTextBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Unicode MS</family>
           </font>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Unicode MS'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.Helvetica Neue DeskInterface';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pauseChecks">
          <item>
           <widget class="QCheckBox" name="pauseScrolling">
            <property name="text">
             <string>Pause Scrolling</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="messageInputLayout">
        <item>
         <layout class="QHBoxLayout" name="messageLabelAndButtons">
          <item>
           <widget class="QLabel" name="serialMessageLabel">
            <property name="text">
             <string>Send as:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sendMessageBit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Bits</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">sendButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sendMessageHex">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Hex</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">sendButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sendMessageChar">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Char</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">sendButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lineEditAndButton">
          <item>
           <widget class="QLineEdit" name="messageLineEdit">
            <property name="font">
             <font>
              <family>Arial Unicode MS</family>
             </font>
            </property>
            <property name="inputMask">
             <string>HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH</string>
            </property>
            <property name="maxLength">
             <number>251</number>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="sendMessage">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>ENTER</string>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerial">
    <property name="title">
     <string>Serial</string>
    </property>
    <addaction name="actionReconnect"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuSerial"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReconnect">
   <property name="text">
    <string>Reconnect</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>messageLineEdit</tabstop>
  <tabstop>sendMessage</tabstop>
  <tabstop>pauseScrolling</tabstop>
  <tabstop>displayOutputBit</tabstop>
  <tabstop>displayOutputHex</tabstop>
  <tabstop>displayOutputChar</tabstop>
  <tabstop>sendMessageHex</tabstop>
  <tabstop>serialTextBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="displayButtonGroup"/>
  <buttongroup name="sendButtonGroup"/>
 </buttongroups>
</ui>
